src/Option/README.md

```diff
@@ -0,0 +1,26 @@
+# Option:
+
+Provides an Option component that is passed as a child to the ListBox component.
+
+## API
+
+### `children` (element array; required)
+`children` specifies the list of elements that will be displayed within the Option component.
+
+### `className` (string array; optional)
+`className` specifies Bootstrap class names to apply to the Option component. The default is an empty array. (do I mention the existing classnames here?)
```

Comment:
Same thing about `className` here that I mentioned in ListBox. Also, I don't think it is necessary to mention the classnames applied to Option components internally by Paragon.